src/reducers/userListsReducer.js

```diff
@@ -1,24 +1,33 @@
 const initialState = {
   liked: [],
+  watchLater: [],
 };
 
 export const userListsReducer = (state = initialState, action) => {
   const { type, payload } = action;
   switch (type) {
-    case "INIT_LIKED":
+    case "INIT_USER_LIST":
+      const { likes, watchlater } = payload;
       return {
         ...state,
-        liked: payload,
+        liked: [...likes],
+        watchLater: [...watchlater],
       };
     case "UPDATE_LIKED":
       return {
         ...state,
         liked: [...payload],
       };
+    case "UPDATE_WATCH_LATER":
+      return {
+        ...state,
+        watchLater: [...payload],
+      };
     case "CLEAR_USER_LISTS":
       return {
         ...state,
         liked: [],
+        watchlater: [],
       };
     default:
       return state;
```

Comment:
Instead of return state, you can also show an error in your default case like
default:
      throw new Error(`Unknown action type: ${action.type}`);

Reply:
Yeah. agree but mostly in production apps we return state only so that it won't break the app with unknown actions.